test/acceptance/trade-area.js

```diff
@@ -57,6 +57,34 @@ describe('trade-area analysis', function() {
             });
         });
 
+        it('should not fail with Postgres reserved words', function (done) {
```

Comment:
we could also add test for column names that contain dashes, spaces, now that we support them